src/synapse/azext_synapse/_params.py

```diff
@@ -19,18 +21,10 @@ def load_arguments(self, _):
     for scope in ['synapse spark batch', 'synapse spark session']:
         with self.argument_context(scope + ' create') as c:
             c.argument('job_name', arg_type=name_type, help='The spark batch or session job name.')
-            c.argument('file', help='The URI of file.')
-            c.argument('class_name', help='The class name.')
-            c.argument('args', nargs='+', help='The arguments of the job.')
-            c.argument('jars', nargs='+', help='The array of jar files.')
-            c.argument('files', nargs='+', help='The array of files URI.')
-            c.argument('archives', nargs='+', help='The array of archives.')
-            c.argument('conf', help='The configuration of spark batch job.')
-            c.argument('driver_memory', help='The memory of driver.')
-            c.argument('driver_cores', help='The number of cores in driver.')
-            c.argument('executor_memory', help='The memory of executor.')
-            c.argument('executor_cores', help='The number of cores in each executor.')
-            c.argument('num_executors', help='The number of executors.')
+            c.argument('reference_files', nargs='+', help='The array of files URI.')

```

Comment:
>The array of files URI. [](start = 59, length = 23)

Nice to use help messages of Synapse Studio job definition page. This is optional.

Reply:
Good idea. Changed.

---
In reply to: [417304288](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r417304288) [](ancestors = 417304288)